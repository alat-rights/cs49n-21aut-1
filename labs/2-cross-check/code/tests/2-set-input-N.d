2-set-input-N.o: 2-set-input-N.c test-helper.h ../rpi.h ../fake-pi.h
