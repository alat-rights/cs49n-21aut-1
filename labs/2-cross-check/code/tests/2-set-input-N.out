TRACE: calling pi code
TRACE: GET32(0x20200000) = 0x327b23c6
TRACE: PUT32(0x20200000) = 0x327b23c0
TRACE: GET32(0x20200000) = 0x327b23c0
TRACE: PUT32(0x20200000) = 0x327b23c0
TRACE: GET32(0x20200000) = 0x327b23c0
TRACE: PUT32(0x20200000) = 0x327b2200
TRACE: GET32(0x20200000) = 0x327b2200
TRACE: PUT32(0x20200000) = 0x327b2000
TRACE: GET32(0x20200000) = 0x327b2000
TRACE: PUT32(0x20200000) = 0x327b0000
TRACE: GET32(0x20200000) = 0x327b0000
TRACE: PUT32(0x20200000) = 0x32780000
TRACE: GET32(0x20200000) = 0x32780000
TRACE: PUT32(0x20200000) = 0x32600000
TRACE: GET32(0x20200000) = 0x32600000
TRACE: PUT32(0x20200000) = 0x32000000
TRACE: GET32(0x20200000) = 0x32000000
TRACE: PUT32(0x20200000) = 0x30000000
TRACE: GET32(0x20200000) = 0x30000000
TRACE: PUT32(0x20200000) = 0x0
TRACE: GET32(0x20200004) = 0x643c9869
TRACE: PUT32(0x20200004) = 0x643c9868
TRACE: GET32(0x20200004) = 0x643c9868
TRACE: PUT32(0x20200004) = 0x643c9840
TRACE: GET32(0x20200004) = 0x643c9840
TRACE: PUT32(0x20200004) = 0x643c9800
TRACE: GET32(0x20200004) = 0x643c9800
TRACE: PUT32(0x20200004) = 0x643c9000
TRACE: GET32(0x20200004) = 0x643c9000
TRACE: PUT32(0x20200004) = 0x643c8000
TRACE: GET32(0x20200004) = 0x643c8000
TRACE: PUT32(0x20200004) = 0x643c0000
TRACE: GET32(0x20200004) = 0x643c0000
TRACE: PUT32(0x20200004) = 0x64200000
TRACE: GET32(0x20200004) = 0x64200000
TRACE: PUT32(0x20200004) = 0x64000000
TRACE: GET32(0x20200004) = 0x64000000
TRACE: PUT32(0x20200004) = 0x60000000
TRACE: GET32(0x20200004) = 0x60000000
TRACE: PUT32(0x20200004) = 0x40000000
TRACE: GET32(0x20200008) = 0x66334873
TRACE: PUT32(0x20200008) = 0x66334870
TRACE: GET32(0x20200008) = 0x66334870
TRACE: PUT32(0x20200008) = 0x66334840
TRACE: GET32(0x20200008) = 0x66334840
TRACE: PUT32(0x20200008) = 0x66334800
TRACE: GET32(0x20200008) = 0x66334800
TRACE: PUT32(0x20200008) = 0x66334000
TRACE: GET32(0x20200008) = 0x66334000
TRACE: PUT32(0x20200008) = 0x66330000
TRACE: GET32(0x20200008) = 0x66330000
TRACE: PUT32(0x20200008) = 0x66300000
TRACE: GET32(0x20200008) = 0x66300000
TRACE: PUT32(0x20200008) = 0x66200000
TRACE: GET32(0x20200008) = 0x66200000
TRACE: PUT32(0x20200008) = 0x66000000
TRACE: GET32(0x20200008) = 0x66000000
TRACE: PUT32(0x20200008) = 0x60000000
TRACE: GET32(0x20200008) = 0x60000000
TRACE: PUT32(0x20200008) = 0x40000000
TRACE: GET32(0x2020000c) = 0x74b0dc51
TRACE: PUT32(0x2020000c) = 0x74b0dc50
TRACE: GET32(0x2020000c) = 0x74b0dc50
TRACE: PUT32(0x2020000c) = 0x74b0dc40
TRACE: pi exited cleanly: 135 calls to random
